bindings/java/iota-client-java/src/test/java/org/iota/HighLevelApiTest.java

```diff
@@ -77,7 +77,7 @@ public void testConsolidateFunds() throws ClientException {
         SecretManager secretManager = new MnemonicSecretManager(DEFAULT_DEVELOPMENT_MNEMONIC);
         String address = client.generateAddresses(secretManager, new GenerateAddressesOptions().withRange(new Range(0, 1)))[0];
         requestFundsFromFaucet(address);
-        String consolidatedAddress = client.consolidateFunds(secretManager, 0, new Range(0, 5));
+        String consolidatedAddress = client.consolidateFunds(secretManager, new GenerateAddressesOptions().withRange(new Range(0, 1)));
```

Comment:
Just checking that the change in range doesn't affect the test itself

Reply:
Just checks less addresses which probably don't have funds anyways, so shouldn't make a difference